### PR TITLE
Email for charge failures (fixes #71)

### LINKED ISSUE #71: Send subscription charge failed email
When braintree tells us a subscription charge failed, send an email looking like this http://payments.readthedocs.org/en/latest/design/api/provider.html#subscription-charge-failed

### PR DESCRIPTION
**Pull request by kumar303**

### CODE REVIEW

File: payments_service/braintree/tests/test_views.py [hunk 1]
```diff
@@ -297,7 +297,7 @@ def test_verify(self):
         eq_(res.status_code, 200)
         eq_(res.content, 'token')
 
-    def test_verify_failed(self):
+    def test_bas_solitude_response_for_verify(self):
```
Comment (andymckay) [inline payments_service/braintree/tests/test_views.py:300]: `bas`?

**Comment by andymckay** [comment]
r+

File: payments_service/braintree/tests/test_views.py (hunk 1 above)
Comment (kumar303) [inline payments_service/braintree/tests/test_views.py:300]: Ha. I don't even know.

Comment (kumar303) [inline payments_service/braintree/tests/test_views.py:300]: fixing it in the next email patch

**Pull request merged**